sdk/internal/perf/status.go

```diff
@@ -164,10 +164,10 @@ func printFinalResults(elapsedTimes [][]float64, perSecondCount [][]int, warmup
 	secondsPerOp := 1.0 / opsPerSecond
 	weightedAvgSec := float64(totalOperations) / opsPerSecond
 	fmt.Printf(
-		"Completed %s operations in a weighted-average of %.2fs (%s ops/s, %.3f s/op)\n",
+		"Completed %s operations in a weighted-average of %ss (%s ops/s, %s s/op)\n",
```

Comment:
This is neat, I'd never heard of message.Printer before.

Reply:
Does a very good job of making numbers more readable with comma separated values.